src/generic/FunctionField.jl

```diff
@@ -562,6 +562,14 @@ Return the characteristic of the underlying rational function field.
 """
 characteristic(R::FunctionField) = characteristic(base_ring(R))
 
+function is_perfect(R::FunctionField)
+  if characteristic(R) == 0
+    return true
+  else
+    return false
+  end
```

Comment:
```suggestion
  return characteristic(R) == 0
  end
```
maybe apply ? but no strong opinion

Reply:
I trusted you

Reply:
Sorry, was on mobile